addons/web/static/src/views/kanban/kanban_column_quick_create.js

```diff
@@ -64,9 +60,10 @@ export class KanbanColumnQuickCreate extends Component {
     }
 
     validate() {
-        if (this.state.columnTitle.length) {
-            this.props.onValidate(this.state.columnTitle);
-            this.state.columnTitle = "";
+        const title = this.inputRef.el.value.trim();
+        if (title.length) {
+            this.props.onValidate(title);
+            this.inputRef.el.value = "";
```

Comment:
This was required because after a column is create, a new quick create column stays open to create another one. Same input, so we need to empty it.